src/schematools/types.py

```diff
@@ -214,11 +227,11 @@ def build_nested_table(
         # we need to add a shortname to the dynamically generated
         # schema definition.
         if field.has_shortname or table.has_shortname:
-            snakecased_fieldname = to_snake_case(field.name)
+            snakecased_fieldname: str = to_snake_case(field.name)
```

Comment:
is dat nodig? niet leiver `to_snake_case()` voorzien van een return type?

Reply:
`to_snake_case()` is already type annotated to return a `str` Hence MyPy should be able to derive this.